src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/RamDeliusClient.kt

```diff
@@ -14,6 +13,6 @@ class RamDeliusClient(private val ramDeliusApiClient: RestClient) {
   fun makePutAppointmentRequest(uri: String, requestBody: Any): AppointmentResponseDTO? =
     ramDeliusApiClient.put(uri, requestBody).retrieve().bodyToMono(AppointmentResponseDTO::class.java).block()
 
-  fun makeGetConvictionRequest(uri: String, token: JwtAuthenticationToken?): ConvictionDetails =
-    ramDeliusApiClient.get(uri, null, token).retrieve().bodyToMono(ConvictionDetails::class.java).block()
+  fun makeGetConvictionRequest(uri: String): ConvictionDetails =
+    ramDeliusApiClient.get(uri, null, null).retrieve().bodyToMono(ConvictionDetails::class.java).block()
```

Comment:
@awhodgson256 The last two parameters are already defaulted to null. So we don't need to pass that as well `ramDeliusApiClient.get(uri, null, null)`

Reply:
@kth13 removed to simplify as suggested